pkg/scaffold/v2/controller/controller_suitetest.go

```diff
@@ -119,12 +134,10 @@ var _ = AfterSuite(func() {
 // adding import paths and code setup for new types.
 func (a *ControllerSuiteTest) Update() error {
 
-	resourcePackage, _ := util.GetResourceInfo(a.Resource, a.Repo, a.Domain)
-
 	ctrlImportCodeFragment := fmt.Sprintf(`"%s/controllers"
 `, a.Repo)
 	apiImportCodeFragment := fmt.Sprintf(`%s%s "%s/%s"
-`, a.Resource.GroupImportSafe, a.Resource.Version, resourcePackage, a.Resource.Version)
```

Comment:
This was introduced in #1219. `ResourcePackage` is only needed at `Update`, that is why it is computed here instead of in `GetInput` and why the field is not needed, just a local variable.

Reply:
Please, see: https://github.com/kubernetes-sigs/kubebuilder/pull/1219/files#r363710055

Reply:
Hi @Adirio, 

It is not a big deal at all. However, the `a.ResourcePackage` is used in this case. You just replace the attribute of the struct for the usage of local var instead of. IMO, the previous code has a better approach in this case in order to keep it done as the conttroller.go as well. However, it is clear that you do not agree with and it is fine.

So, if @mengqiy also prefers as you keep this part as done in #1219 + changes I will do that. Otherwise, if @mengqiy be ok with and see that is no required perform changes here I leave it as it is done, OK?

Reply:
The thing is that `controller.go` does require it:

https://github.com/kubernetes-sigs/kubebuilder/blob/cf6cec42bdf594eac67cfb8f13eee6f57ba5c47b/pkg/scaffold/v2/controller.go#L79

You can see how `main.go` doesn't need it (same as in this case) and thus just uses a local variable:

https://github.com/kubernetes-sigs/kubebuilder/blob/cf6cec42bdf594eac67cfb8f13eee6f57ba5c47b/pkg/scaffold/v2/main.go#L56

Reply:
Done as you requested. 👍 
Marked as solved.